src/main.cpp

```diff
@@ -125,26 +123,31 @@ path_finder_test(const std::string path, std::string save_path)
     std::pair<terrain::Tile*, terrain::Tile*> start_end =
         world.terrain_main.get_start_end_test();
 
-    std::cout << "Start: " << start_end.first->get_x() << ", "
-              << start_end.first->get_y() << ", " << start_end.first->get_z()
-              << std::endl;
-    std::cout << "End:   " << start_end.second->get_x() << ", "
-              << start_end.second->get_y() << ", " << start_end.second->get_z()
-              << std::endl;
+    LOG_INFO(
+        logger, "Start: {}, {}, {}", start_end.first->get_x(), start_end.first->get_y(),
+        start_end.first->get_z()
+    );
+
+    LOG_INFO(
+        logger, "End: {}, {}, {}", start_end.second->get_x(), start_end.second->get_y(),
+        start_end.second->get_z()
+    );
 
     std::vector<const terrain::Tile*> tile_path =
         world.terrain_main.get_path_Astar(start_end.first, start_end.second);
 
-    std::cout << "    " << static_cast<int>(tile_path.size()) << std::endl;
+    LOG_INFO(
+        logger, "Path length: {}", static_cast<int>(tile_path.size())
+    );
+
     if (tile_path.size() == 0) {
-        std::cout << "no path" << std::endl;
+        LOG_INFO(
+            logger, "No path");
```

Comment:
Format

And this should be a warning

Reply:
why? If you ask the computer for a path to the moon, and its says no path is that a warning?

Reply:
Yes because that would not be expected behavior